### DISCUSSION TITLE
[KDE] How to change Caps Lock to normal functionality?

### DISCUSSION THREAD

**Discussion by birds-swim** [SpiralLinux/SpiralLinux-project#259] [Q&A] [answered]
Hello again,

I'm setting up another computer for a family member, but this time with KDE Plasma.  I am trying to change the Caps Lock to normal behavior as I do not need support for multiple languages.  I'm looking in KDE's System Settings app, but I can't find the setting under the Keyboard section.  How do I change my Caps Lock to be like a regular keyboard?

**Comment by geckolinux** [comment, marked as answer]
![](https://i.imgur.com/9iiRcXk.png)

**Reply by birds-swim** [reply]
Thank you!
